### PR TITLE
Add request console server action

### PR DESCRIPTION
**Pull request by TahaPY**
Added option to Use this package for to multi API key & I add console… func to get console URL
please check it if is a good option merge it to base repo
thanks for this good package

### CODE REVIEW

File: hcloud/schema.go [hunk 1]
```diff
@@ -124,7 +124,7 @@ func DatacenterFromSchema(s schema.Datacenter) *Datacenter {
 }
 
 // ServerFromSchema converts a schema.Server to a Server.
-func ServerFromSchema(s schema.Server) *Server {
```
Comment (LKaemmerling) [inline hcloud/schema.go]: There is no need to add the `ServersClient` here.

File: hcloud/schema/server.go [hunk 2]
```diff
@@ -139,6 +139,14 @@ type ServerActionRebootResponse struct {
 	Action Action `json:"action"`
 }
 
+// ServerActionConsoleResponse defines the schema of the response when
+// creating a reboot server action.
```
Comment (LKaemmerling) [inline hcloud/schema/server.go]: Should be a sentence about requestion the console.

File: hcloud/server.go [hunk 3]
```diff
@@ -399,6 +400,22 @@ func (c *ServerClient) Reboot(ctx context.Context, server *Server) (*Action, *Re
 	return ActionFromSchema(respBody.Action), resp, nil
 }
 
+// Console get console of a server.
```
Comment (LKaemmerling) [inline hcloud/server.go]:
```suggestion
// Console requesrs a VNC console for a server.
```

File: hcloud/schema.go (hunk 1 above)
Comment (TahaPY) [reply hcloud/schema.go]: I added this for use 2 API key or more 
I need this but if you have a better solution say

Comment (LKaemmerling) [reply hcloud/schema.go]: A better solution would be to just use two client instances.

Comment (TahaPY) [reply hcloud/schema.go]: `var servers = make(map[string]*hcloud.Server)`
I have this but I can't reboot or sth else with this
I added `ServerClient` to `Server` for this reason
but ok I delete that for merge console request
if you have any idea for use 2 or more API key plz say

Comment (LKaemmerling) [reply hcloud/schema.go]: You could just use two Client instances. In our mind, it does not make sense to add a client here:
```go
client1 := hcloud.NewClient(hcloud.WithToken("token1"))
client2 := hcloud.NewClient(hcloud.WithToken("token2"))
```

**Comment by TahaPY** [comment]
hello can you merge this pull request to add console request ??

**Comment by LKaemmerling** [comment]
Hello,

you have closed this PR so we assumed that you are not interested anymore.  I have requested some changes before these changes are not applied I can not merge this PR.

**Pull request closed without merging**